src/deriver/queue_manager.py

```
@@ -117,37 +116,73 @@
+        stmt = (
+            insert(models.ActiveQueueSession)
+            .values(values)
+            .on_conflict_do_nothing()
+            .returning(models.ActiveQueueSession.work_unit_key)
         )
 
-        result = await db.execute(query)
-        return result.scalars().all()
+        result = await db.execute(stmt)
+        claimed_units = result.scalars().all()
+        logger.debug(f"Claimed {len(claimed_units)} work units")
```

Ruff (0.12.2)

Logging statement uses f-string

(G004)